schemas/dayDescription.ts

```diff
@@ -0,0 +1,38 @@
+export default {
+  name: "dayDescription",
+  type: "document",
+  title: "Day Description",
+  preview: {
+    select: {
+      dayOfWeek: "dayOfWeek",
+      date: "date",
+    },
+    prepare(selection: Record<string, any>) {
+      const { dayOfWeek, date } = selection;
+
+      return {
+        title:
+          dayOfWeek +
+          (date ? ` ${date.split("-")[2]}.${date.split("-")[1]}` : ""),
+      };
+    },
+  },
+  fields: [
+    {
+      name: "dayOfWeek",
+      type: "string",
+      title: "Dag i uken",
+    },
```

Comment:
Can't this be extracted from the date as is done with the events?

Reply:
True, fixed:P